### ISSUE TITLE
Scrollable outputs are not keyboard-accessible

### ISSUE THREAD

**Issue by choldgraf** [executablebooks/thebe#794]
thebe-core makes cell outputs scrollable but not focusable (e.g. `tabIndex=0`),  so keyboard users can't scroll wide outputs (e.g. pandas DataFrames, wide plotly plots).

I think the problem is that this sets an overflow, but we don't set `tabIndex` etc on the elements that it applies to:

https://github.com/jupyter-book/thebe/blob/c69aae7acaab7901175532635ac7c530df335025/packages/core/src/index.css#L74-L77

### How to fix

I think we'd need to dynamically sense whether the output is going to overflow, and apply `tabIndex=0` and role="region" etc if it does, either with a MutationObserver or ResizeObserver.

### Notes

- We first ran into this with @pancakereport's axe checker via https://github.com/jupyter-book/myst-theme/pull/864
